pyvista/plotting/tools.py

```diff
@@ -330,12 +331,17 @@ def opacity_transfer_function(mapping, n_colors, interpolate=True,
                     raise ValueError('No interpolation.')
                 # Use a quadratic interp if scipy is available
                 from scipy.interpolate import interp1d
+
                 # quadratic has best/smoothest results
                 f = interp1d(xo, mapping, kind=kind)
                 vals = f(xx)
                 vals[vals < 0] = 0.0
                 vals[vals > 1.0] = 1.0
                 mapping = (vals * 255.).astype(np.uint8)
+                # nasty issue on Python 3.9 with 9.0.20210612.dev0
+                if sys.version_info.minor == 9:
+                    import gc; gc.collect()
+
```

Comment:
Coming back to earlier comments: are you positive this is necessary on 3.9, and only on 3.9? Can you pinpoint why this is needed or is there some magic involved? If we anticipate a given kind of reference cycle we might want to be more specific in the comment, for instance.

I'm just a bit worried that this will stick around and eventually nobody will know what it's for and when it can be removed.

Reply:
Removed.  Bug went away on its own after a restart; which scares me.

I have no idea why VTK is so unstable.